components/HomeHero/HomeHero.tsx

```diff
@@ -0,0 +1,28 @@
+import Image from 'next/image';
+
+import logo from '../../public/images/headless-logo.svg';
+
+import styles from './HomeHero.module.scss';
+
+import Strapline from './Strapline';
+import type { StraplineProps } from './Strapline/types';
+
+const HomeHero = ({ icon, text }: StraplineProps) => (
+    <div className={styles.hero}>
+        <div className={styles.logo}>
+            <Image src={logo} alt="" />
+        </div>
+        <div className={styles.hero__body}>
+            <h1 className={styles.title}>
+                Are we <br />
+                <strong className={styles.title__emphasis}>
+                    headless
+                </strong>{' '}
+                yet?
+            </h1>
+            <Strapline icon={icon} text={text} />
```

Comment:
Yes, but the strapline component comes with its own types and style. It would only add 8 lines in the `HomeHero.tsx` file but 31 more in the styles file.

Reply:
That's not too bad, but the class names would have to be slightly more specific than if you just stick with what you have. I might be biased by long SCSS files with lots of BEM child classes